docs/index.md

```diff
@@ -15,1621 +15,118 @@ parent="smn_workw_machine"
-command again:
-
-```
-$ docker-machine ls
-NAME   ACTIVE   DRIVER       STATE     URL                         SWARM
-dev             virtualbox   Running   tcp://192.168.99.100:2376
-```
+Docker Machine supports over fourteen providers including Amazon Web Services,
+Azure, Digital Ocean, and more. It also supports a generic provider if you want
+to add your own provider.
 
-Next, as noted in the output of the `docker-machine create` command, we have to
-tell Docker to talk to that machine. You can do this with the `docker-machine
-env` command. For example,
+After Machine has your Docker Host up and running, the second step is to
+direct your command-line shell to work with your new machine:
 
+```bash
+$ eval "$(docker-machine env my-docker-machine)"
 ```
-$ eval "$(docker-machine env dev)"
-$ docker ps
-```
-
-> **Note**: If you are using `fish`, or a Windows shell such as
-> Powershell/`cmd.exe` the above method will not work as described. Instead,
-> see [the `env` command's documentation](https://docs.docker.com/machine/#env)
-> to learn how to set the environment variables for your shell.
 
-This will set environment variables that the Docker client will read which specify
-the TLS settings. Note that you will need to do that every time you open a new tab or
-restart your machine.
+This command sets a number of variables:
 
-To see what will be set, run `docker-machine env dev`.
-
-```
-$ docker-machine env dev
+```bash
 export DOCKER_TLS_VERIFY="1"
-export DOCKER_HOST="tcp://172.16.62.130:2376"
-export DOCKER_CERT_PATH="/Users/<your username>/.docker/machine/machines/dev"
-export DOCKER_MACHINE_NAME="dev"
-# Run this command to configure your shell:
-# eval "$(docker-machine env dev)"
+export DOCKER_HOST="tcp://192.168.99.100:2376"
+export DOCKER_CERT_PATH="/Users/mary/.docker/machine/machines/my-docker-machine"
+export DOCKER_MACHINE_NAME="my-docker-machine"
 ```
 
-You can now run Docker commands on this host:
+The `DOCKER_TLS_VERIFY` sets transport layer security (TLS) on communication
+between your machine and the `DOCKER_HOST` machine. TLS is a protocol to allow
+client/server applications to communicate over the Internet securely. The
+`DOCKER_CERT_PATH` is the path to the certificate.  
 
-```
-$ docker run busybox echo hello world
-Unable to find image 'busybox' locally
-Pulling repository busybox
-e72ac664f4f0: Download complete
-511136ea3c5a: Download complete
-df7546f9f060: Download complete
-e433a6c5b276: Download complete
-hello world
-```
-
-Any exposed ports are available on the Docker host’s IP address, which you can
-get using the `docker-machine ip` command:
-
-```
-$ docker-machine ip dev
-192.168.99.100
-```
-
-For instance, you can try running a webserver ([nginx](https://nginx.org)) in a
-container with the following command:
+Once you've configured your shell, the third step is use the Docker Engine `docker`
+command line commands to download images and run containers. For example, to
+start an `nginx` host:
```

Comment:
The use of "host" here seems confusing, since we've just created a "host".  It's an `nginx` container, we should probably call it that.